tests/lib/rules/type-annotation-spacing.js

```diff
@@ -4234,11 +4229,6 @@ type Bar = Record<keyof Foo, string>
             ],
             parser: "typescript-eslint-parser",
         },
-        `
-interface resolve {
-    resolver?: (() => PromiseLike<T>) | PromiseLike<T>;
-}
-        `,
```

Comment:
Why the removed tests?

Reply:
this test is duplicated 3 times

Reply:
https://github.com/bradzacher/eslint-plugin-typescript/blob/master/tests/lib/rules/type-annotation-spacing.js#L25
https://github.com/bradzacher/eslint-plugin-typescript/blob/master/tests/lib/rules/type-annotation-spacing.js#L3458
https://github.com/bradzacher/eslint-plugin-typescript/blob/master/tests/lib/rules/type-annotation-spacing.js#L4237